e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/A_PublishSingleNodeOrchestratedTestTheory.cs

```diff
@@ -14,56 +14,58 @@ namespace IIoTPlatform_E2E_Tests.Orchestrated
-    [Collection("IIoT Platform Test Collection")]
+    [TestCaseOrderer(TestCaseOrderer.FullName, TestConstants.TestAssemblyName)]
+    [Collection("IIoT Multiple Nodes Test Collection")]
     [Trait(TestConstants.TraitConstants.PublisherModeTraitName, TestConstants.TraitConstants.PublisherModeOrchestratedTraitValue)]
-    public class PublishSingleNodeOrchestratedTestTheory
+    public class A_PublishSingleNodeOrchestratedTestTheory
     {
         private readonly ITestOutputHelper _output;
-        private readonly IIoTPlatformTestContext _context;
+        private readonly IIoTMultipleNodesTestContext _context;
 
-        public PublishSingleNodeOrchestratedTestTheory(IIoTPlatformTestContext context, ITestOutputHelper output) {
+        public A_PublishSingleNodeOrchestratedTestTheory(IIoTMultipleNodesTestContext context, ITestOutputHelper output) {
             _output = output ?? throw new ArgumentNullException(nameof(output));
             _context = context ?? throw new ArgumentNullException(nameof(context));
             _context.OutputHelper = _output;
         }
 
         [Fact, PriorityOrder(0)]
-        public async Task Test_SetUnmanagedTagFalse() {
-            await TestHelper.SwitchToOrchestratedModeAsync(_context);
+        public void Test_SetUnmanagedTagFalse() {
+            _context.Reset();
+            TestHelper.SwitchToOrchestratedModeAsync(_context).GetAwaiter().GetResult();
         }
 
         [Fact, PriorityOrder(1)]
-        public async Task Test_CollectOAuthToken() {
+        public void Test_CollectOAuthToken() {
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-            var token = await TestHelper.GetTokenAsync(_context, cts.Token);
+            var token = TestHelper.GetTokenAsync(_context, cts.Token).GetAwaiter().GetResult();
             Assert.NotEmpty(token);
         }
 
         [Fact, PriorityOrder(2)]
-        public async Task Test_ReadSimulatedOpcUaNodes() {
+        public void Test_ReadSimulatedOpcUaNodes() {
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-            var simulatedOpcServer = await TestHelper.GetSimulatedPublishedNodesConfigurationAsync(_context, cts.Token);
+            var simulatedOpcServer = TestHelper.GetSimulatedPublishedNodesConfigurationAsync(_context, cts.Token).GetAwaiter().GetResult();
             Assert.NotNull(simulatedOpcServer);
             Assert.NotEmpty(simulatedOpcServer.Keys);
             Assert.NotEmpty(simulatedOpcServer.Values);
         }
 
         [Fact, PriorityOrder(3)]
-        public async Task Test_RegisterOPCServer_Expect_Success() {
+        public void Test_RegisterOPCServer_Expect_Success() {
 
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
 
             // We will wait for microservices of IIoT platform to be healthy and modules to be deployed.
```

Comment:
>We will wait  [](start = 15, length = 13)

in general its better to use a distant technical language than first person chat for comments

Reply:
comment is not new, will be changed in further version